lib/packages/core/src/constants/brsVersion.ts

```diff
@@ -0,0 +1,4 @@
+export enum BrsVersion {
```

Comment:
naive solution...but sufficient for MVP

Reply:
Does anything in the angular wallet need to be modified for this to work?